

<!DOCTYPE html>

<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.18.1: http://docutils.sourceforge.net/" />

<title>Curriculum Vitae - Korean</title>

    <link rel="stylesheet" type="text/css" href="_static/pygments.css" />
    <link rel="stylesheet" type="text/css" href="_static/main.css" />
    <script data-url_root="./" id="documentation_options" src="_static/documentation_options.js"></script>
    <script src="_static/doctools.js"></script>
    <script src="_static/sphinx_highlight.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" /> 
  </head><body>


      <div class="documentwrapper">
        <div class="bodywrapper">
          <div class="body" role="main">
            
  <section class="page text-center" id="dr-jeongbin-park">
<h1>Dr. Jeongbin Park</h1>
<p>(마지막 갱신일: Apr 25, 2023)</p>
<div class="footercv docutils container">
<p>(이 CV의 최신 버전은 여기에서 확인하실 수 있습니다: <a class="reference external" href="https://jeongbinpark.com/cv-ko">https://jeongbinpark.com/cv-ko</a>)</p>
</div>
<div class="small docutils container">
<div class="line-block">
<div class="line">주소: 경남 양산시 물금읍 부산대학로 49 첨단의생명융합센터 208-1호 (우: 50612)</div>
<div class="line">이메일: <a class="reference external" href="mailto:jeongbin&#46;park&#37;&#52;&#48;pusan&#46;ac&#46;kr">jeongbin<span>&#46;</span>park<span>&#64;</span>pusan<span>&#46;</span>ac<span>&#46;</span>kr</a></div>
<div class="line">전화: 051-510-8597</div>
<div class="line"><br /></div>
</div>
</div>
<section id="education">
<h2>EDUCATION</h2>
<table class="docutils align-default">
<colgroup>
<col style="width: 25%" />
<col style="width: 75%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><p>2016 - 2020</p></td>
<td><p><strong>이학박사 (Dr. rer. nat.)</strong>, 생물학, 하이델베르크 대학교, 독일</p></td>
</tr>
<tr class="row-even"><td><p>2012 - 2014</p></td>
<td><p><strong>이학석사 (M. Sc.)</strong>, 물리학, 서울대학교, 대한민국</p></td>
</tr>
<tr class="row-odd"><td><p>2008 - 2012</p></td>
<td><p><strong>이학사 (B. Sc.)</strong>, 물리학, 부산대학교, 대한민국</p></td>
</tr>
</tbody>
</table>
</section>
<section id="work-experience">
<h2>WORK EXPERIENCE</h2>
<table class="docutils align-default">
<colgroup>
<col style="width: 25%" />
<col style="width: 75%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><p>2022 - 현재</p></td>
<td><div class="line-block">
<div class="line"><strong>조교수</strong>, 의생명융합공학부, 부산대학교</div>
</div>
</td>
</tr>
<tr class="row-even"><td><p>2020 - 2022</p></td>
<td><div class="line-block">
<div class="line"><strong>박사후 과정 연구원</strong>, Division of Computational Genomics and System Genetics, 독일 암 연구소 (DKFZ)</div>
<div class="line">(지도교수: Prof. Oliver Stegle)</div>
</div>
</td>
</tr>
<tr class="row-odd"><td><p>2018 - 2020</p></td>
<td><div class="line-block">
<div class="line"><strong>박사과정 연구원</strong>, Digital Health Center, Berlin Institute of Health (BIH) 및 샤리테 (Charité) 대학병원</div>
<div class="line">(지도교수: Prof. Roland Eils)</div>
</div>
</td>
</tr>
<tr class="row-even"><td><p>2016 - 2018</p></td>
<td><div class="line-block">
<div class="line"><strong>박사과정 연구원</strong>, Division of Theoretical Bioinformatics, 독일 암 연구소 (DKFZ)</div>
<div class="line">(지도교수: Prof. Roland Eils)</div>
</div>
</td>
</tr>
<tr class="row-odd"><td><p>2015 - 2016</p></td>
<td><div class="line-block">
<div class="line"><strong>공동연구</strong>, 분자유전공학연구실, 한양대학교</div>
<div class="line">(공동연구: 배상수 교수)</div>
</div>
</td>
</tr>
<tr class="row-even"><td><p>2014 - 2015</p></td>
<td><div class="line-block">
<div class="line"><strong>공동연구</strong>, 유전체교정연구단, 서울대학교 및 기초과학연구원 (IBS)</div>
<div class="line">(공동연구: 배상수 박사)</div>
</div>
</td>
</tr>
<tr class="row-odd"><td><p>2012 - 2014</p></td>
<td><div class="line-block">
<div class="line"><strong>석사과정 연구원</strong>, 단분자생물물리연구실, 서울대학교</div>
<div class="line">(지도교수: 홍성철 교수)</div>
</div>
</td>
</tr>
</tbody>
</table>
</section>
<section id="service">
<h2>SERVICE</h2>
<div class="entries text-left docutils container">
<p><strong>학회 활동:</strong></p>
<ul class="simple">
<li><p>한국유전체학회 정보전산위원, 2023</p></li>
<li><p>한국생명정보학회 대외위원, 2023</p></li>
</ul>
<p><strong>Ad-hoc 저널 리뷰어:</strong></p>
<ul class="simple">
<li><p>Bioinformatics, NAR: Genomics and Bioinformatics, Scientific Reports, BMC Genomics, Journal of Open Source Software (JOSS)</p></li>
</ul>
<p><strong>번역:</strong></p>
<ul class="simple">
<li><p><strong>A Byte of Python (한국어 번역)</strong>: 온라인 파이썬 가이드북 (원저자: Swaroop C H). 링크: <a class="reference external" href="https://github.com/pjb7687/byte_of_python/">https://github.com/pjb7687/byte_of_python/</a></p></li>
</ul>
<p><strong>오픈소스 프로젝트:</strong></p>
<ul class="simple">
<li><p><strong>GNU Data Language (GDL)</strong>: GDL 프로젝트 (<a class="reference external" href="https://github.com/gnudatalanguage/gdl/">https://github.com/gnudatalanguage/gdl/</a>) 윈도우 버전 포팅에 기여함 (2014 - 2015년). 2021년부터 프로젝트 Release Manager로 기여하고 있음.</p></li>
</ul>
</div>
</section>
<section id="teaching-experience">
<h2>TEACHING EXPERIENCE</h2>
<div class="entries text-left docutils container">
<p><strong>부산대학교</strong>:</p>
<ul class="simple">
<li><p><strong>“정보의학개론”</strong>, <strong>“유닉스 기초”</strong>, <strong>“객체지향프로그래밍”</strong>, 부산대학교 의생명융합공학부</p></li>
<li><p><strong>“임상및유전체정보학”</strong>, 부산대학교 대학원 유전체데이터과학전공</p></li>
</ul>
<p><strong>부산대학교 이전:</strong></p>
<p>초청강연 <strong>“Spatial transcriptomics”</strong>, lecture series “Single cell RNA-seq data analysis with R” (<strong>ELIXIR EXCELERATE</strong> 프로젝트). CSC, Espoo, 핀란드. 2019년 5월 29일.</p>
<p>학생 지도, 연구 인턴 3인 지도 (공동지도 1인 포함), Division of Theoretical Bioinformatics, 독일 암 연구소 (DKFZ), 독일. 2016년 7월 - 2018년 2월.</p>
<p>강의조교:</p>
<ul class="simple">
<li><p><strong>“Methoden der Bioinformatik - Python”</strong> (강의보조), Faculty of Biosciences, 하이델베르크 대학교, WS 2016 및 WS 2017.</p></li>
<li><p><strong>“물리학 실험”</strong>, 물리학과, 서울대학교, SS 2012.</p></li>
</ul>
</div>
</section>
<section id="conference-proceedings">
<h2>CONFERENCE PROCEEDINGS</h2>
<div class="entries text-justify publications docutils container">
<p><a class="reference external" href="https://cancerres.aacrjournals.org/content/80/16_Supplement/1491.short">Martina K Zowada, Stephan M Tirier, Sebastian M Dieter, Teresa Krieger, Ava Oberlack, Robert L Chua, Mario Huerta, Foo Wei Ten, Karin Laaber, <span class="underline">Jeongbin Park</span>, Katharina Jechow, Torsten Müller, Mathias Kalxdorf, Mark Kriegsmann, Katharina Kriegsmann, Friederike Herbst, Jeroen Krijgsveld, Martin Schneider, Roland Eils, Hanno Glimm, Christian Conrad, Claudia R Ball, <strong>“Transcriptional heterogeneity identifies functional states of tumor-initiating cell differentiation in human colorectal cancer”</strong>, <em>Cancer Research</em> 80 (16 Supplement), Abstract nr. 1491 (2020).</a></p>
<p><a class="reference external" href="https://onlinelibrary.wiley.com/doi/10.1111/bjh.15536">Cristina Lopez, Rabea Wagener, Kortine Kleinheinz, Julia Bausinger, Sietse M Aukema, Inga Nagel, Umut H Toprak, Julian Seufert, Janine Altmueller, Holger Thiele, Christof Schneider, Julia Kolarova, <span class="underline">Jeongbin Park</span>, Daniel Huebschmann, Eva M Murga Penas, Hans G Drexler, Andishe Attarbaschi, Randi Hovland, Eigil Kjeldsen, Michael Kneba, Udo Kontny, Laurence de Leval, Peter Nuernberg, Ilske Oschlies, David Oscier, Brigitte Schlegelberger, Stephan Stilgenbauer, Wilhelm Woessmann, Matthias Schlesner, Birgit Burkhardt, Wolfram Klapper, Elaine S Jaffe, Ralf Kueppers, Reiner Siebert, <strong>“IG-MYC-positive leukemia and lymphoma with precursor B-cell phenotype are genetically and epigenetically distinct from Burkitt lymphomas”</strong>, <em>British Journal of Haematology</em> 182 (S1), 19-21, Abstract nr. 23 (2018).</a></p>
<p><a class="reference external" href="https://ui.adsabs.harvard.edu/abs/2014ASPC..485..331C/abstract">Alain Coulais, Marc Schellens, Gilles Duvert, <span class="underline">Jeongbin Park</span>, Orion Poplawski, Sylwester Arabas, Stephane Erard, Benjamin Laurent, Mathieu Pinter, Nodar Kasradze, Mouadh Ayad, <strong>“Scaling GDL for multi-cores to process Planck HFI beams Monte Carlo on HPC”</strong>, <em>Astronomical Data Analysis Software and Systems XXIII</em> 485, 331 (2014). [cited: 2]</a></p>
</div>
</section>
<section id="publications">
<h2>PUBLICATIONS</h2>
<div class="entries text-justify publications docutils container">
<p><a class="reference external" href="https://genominfo.org/upload/pdf/gi-22079.pdf"><span class="underline">Jeongbin Park</span>, Nagarajan Paramasivam, <strong>“PyOncoPrint: a python package for plotting OncoPrints”</strong>, <em>Genomics &amp; Informatics</em> 21 (1), e14 (2023).</a></p>
<p>Brian Long, Jeremy Miller, The SpaceTx Consortium, <strong>“SpaceTx: A Roadmap for Benchmarking Spatial Transcriptomics Exploration of the Brain”</strong>, <em>arXiv</em>, 2301.08436 (2023). [cited: 1]</p>
<p>Seul Lee, Taein Kim, Min-Soo Choi, Yejin Kwak, <span class="underline">Jeongbin Park</span>, Sung Ju Hwang, Sang-Gyu Kim, <strong>“READRetro: Natural Product Biosynthesis Planning with Retrieval-Augmented Dual-View Retrosynthesis”</strong>, <em>bioRxiv</em>, 2023.03. 21.533616 (2023).</p>
<p><a class="reference external" href="https://joss.theoj.org/papers/10.21105/joss.04633"><span class="underline">Jeongbin Park</span>, Gilles Duvert, Alain Coulais, Gregory V Jung, Sylwester Arabas, Brian Barker, Takeshi Enomoto, Sylvain Flinois, Oliver Gressel, Tomas Hillberg, Thibault Huillet, Jan Kohnert, Orion Poplawski, Eloi Rozier de Linage, Remi A Solås, Luke Stagner, Ole Streicher, James Tappin, Thierry Thomas, Jingwei Wang, Christian Wimmer, <strong>“GNU Data Language 1.0: a free/libre and open-source drop-in replacement for IDL/PV-WAVE”</strong>, <em>Journal of Open Source Software</em> 7 (80), 4633 (2022).</a></p>
<p><a class="reference external" href="https://doi.org/10.1101/2022.03.28.486139">Yun Zhang<sup>†</sup>, Jeremy Andrew Miller<sup>†</sup>, <span class="underline">Jeongbin Park</span><sup>†</sup>, Boudewijn P Lelieveldt, Brian D Aevermann, Tommaso Biancalani, Charles Comiter, Christoffer Mattsson Langseth, Brian Long, Viktor Petukhov, Gabriele Scalia, Eeshit Dhaval Vaishnav, Yilin Zhao, Ed S Lein, Richard H Scheuermann, <strong>“Reference-based cell type matching of spatial transcriptomics data”</strong>, <em>bioRxiv</em> (2022). [cited: 2]</a></p>
<p><a class="reference external" href="https://www.nature.com/articles/s41467-021-23807-4"><span class="underline">Jeongbin Park</span><sup>†</sup>, Wonyl Choi<sup>†</sup>, Sebastian Tiesmeyer, Brian Long, Lars E Borm, Emma Garren, Thuc Nghi Nguyen, Bosiljka Tasic, Simone Codeluppi, Tobias Graf, Matthias Schlesner, Oliver Stegle, Roland Eils, Naveed Ishaque, <strong>“Cell segmentation-free inference of cell types from in situ transcriptomics data”</strong>, <em>Nature Communications</em> 12, 3545 (2021). [cited: 58]</a></p>
<p><a class="reference external" href="https://www.mdpi.com/1020628">Martina K Zowada, Stephan M Tirier, Sebastian M Dieter, Teresa G Krieger, Ava Oberlack, Robert Lorenz Chua, Mario Huerta, Foo Wei Ten, Karin Laaber, <span class="underline">Jeongbin Park</span>, Katharina Jechow, Torsten Müller, Mathias Kalxdorf, Mark Kriegsmann, Katharina Kriegsmann, Friederike Herbst, Jeroen Krijgsveld, Martin Schneider, Roland Eils, Hanno Glimm, Christian Conrad, Claudia R Ball, <strong>“Functional states in tumor-initiating cell differentiation in human colorectal cancer”</strong>, <em>Cancers</em> 13 (5), 1097 (2021). [cited: 7]</a></p>
<p><a class="reference external" href="https://academic.oup.com/neuro-oncology/article-abstract/22/8/1138/5820825">Teresa G Krieger, Stephan M Tirier, <span class="underline">Jeongbin Park</span>, Katharina Jechow, Tanja Eisemann, Heike Peterziel, Peter Angel, Roland Eils, Christian Conrad, <strong>“Modeling glioblastoma invasion using human brain organoids and single-cell transcriptomics”</strong>, <em>Neuro-oncology</em>, noaa091 (2020). [cited: 72]</a></p>
<p>Andres Quintero, Daniel Hübschmann, Nils Kurzawa, Sebastian Steinhauser, Philipp Rentzsch, Stephen Krämer, Carolin Andresen, <span class="underline">Jeongbin Park</span>, Roland Eils, Matthias Schlesner, Carl Herrmann, <strong>“ShinyButchR: interactive NMF-based decomposition workflow of genome-scale datasets”</strong>, <em>Biology Methods and Protocols</em> 5 (1), bpaa022 (2020). [cited: 11]</p>
<p><a class="reference external" href="https://www.nature.com/articles/s41598-019-48771-4">Stephan M Tirier, <span class="underline">Jeongbin Park</span>, Friedrich Preußer, Lisa Amrhein, Zuguang Gu, Simon Steiger, Jan-Philipp Mallm, Teresa Krieger, Marcel Waschow, Björn Eismann, Marta Gut, Ivo G Gut, Karsten Rippe, Matthias Schlesner, Fabian Theis, Christiane Fuchs, Claudia R Ball, Hanno Glimm, Roland Eils, Christian Conrad, <strong>“pheno-seq–linking visual features and gene expression in 3D cell culture systems”</strong>, <em>Scientific Reports</em> 9, 12367 (2019). [cited: 18]</a></p>
<p><a class="reference external" href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2585-4?optIn=true">Gue-Ho Hwang<sup>†</sup>, <span class="underline">Jeongbin Park</span><sup>†</sup>, Kayeong Lim, Sunghyun Kim, Jihyeon Yu, Eunchong Yu, Sang-Tae Kim, Roland Eils, Jin-Soo Kim, Sangsu Bae, <strong>“Web-based design and analysis tools for CRISPR base editing”</strong>, <em>BMC Bioinformatics</em> 19, 542 (2018). [cited: 119]</a></p>
<p><a class="reference external" href="https://ashpublications.org/blood/article-abstract/132/21/2280/39527">Rabea Wagener, Cristina López, Kortine Kleinheinz, Julia Bausinger, Sietse M Aukema, Inga Nagel, Umut H Toprak, Julian Seufert, Janine Altmüller, Holger Thiele, Christof Schneider, Julia Kolarova, <span class="underline">Jeongbin Park</span>, Daniel Hübschmann, Eva M Murga Penas, Hans G Drexler, Andishe Attarbaschi, Randi Hovland, Eigil Kjeldsen, Michael Kneba, Udo Kontny, Laurence De Leval, Peter Nürnberg, Ilske Oschlies, David Oscier, Brigitte Schlegelberger, Stephan Stilgenbauer, Wilhelm Wössmann, Matthias Schlesner, Birgit Burkhardt, Wolfram Klapper, Elaine S Jaffe, Ralf Küppers, Reiner Siebert, <strong>“IG-MYC+ neoplasms with precursor B-cell phenotype are molecularly distinct from Burkitt lymphomas”</strong>, <em>Blood</em> 132 (21), 2280-2285 (2018). [cited: 44]</a></p>
<p><a class="reference external" href="https://www.biorxiv.org/content/10.1101/397935v1.abstract"><span class="underline">Jeongbin Park</span><sup>†</sup>, Liam H Childs<sup>†</sup>, Matthias Schlesner, Roland Eils, <strong>“HTSlib-js: developer toolkit for a client-side JavaScript interface for HTSlib”</strong>, <em>bioRxiv</em>, 397935 (2018). [cited: 1]</a></p>
<p><a class="reference external" href="https://www.nature.com/articles/nmeth.4554">Sang-Tae Kim<sup>†</sup>, <span class="underline">Jeongbin Park</span><sup>†</sup>, Daesik Kim, Kyoungmi Kim, Sangsu Bae, Matthias Schlesner, Jin-Soo Kim, <strong>“Response to “Unexpected mutations after CRISPR–Cas9 editing in vivo””</strong>, <em>Nature Methods</em> 15 (4), 239-240 (2018). [cited: 48]</a></p>
<p><a class="reference external" href="https://academic.oup.com/bioinformatics/article-abstract/34/6/1077/4665414"><span class="underline">Jeongbin Park</span>, Sangsu Bae, <strong>“Cpf1-Database: web-based genome-wide guide RNA library design for gene knockout screens using CRISPR-Cpf1”</strong>, <em>Bioinformatics</em> 34 (6), 1077-1079 (2018). [cited: 26]</a></p>
<p><a class="reference external" href="https://www.nature.com/articles/s41467-018-03025-1">Manuel Rodríguez-Paredes, Felix Bormann, Günter Raddatz, Julian Gutekunst, Carlota Lucena-Porcel, Florian Köhler, Elisabeth Wurzer, Katrin Schmidt, Stefan Gallinat, Horst Wenck, Joachim Röwert-Huber, Evgeniya Denisova, Lars Feuerbach, <span class="underline">Jeongbin Park</span>, Benedikt Brors, Esther Herpel, Ingo Nindl, Thomas G Hofmann, Marc Winnefeld, Frank Lyko, <strong>“Methylation profiling identifies two subclasses of squamous cell carcinoma related to distinct cells of origin”</strong>, <em>Nature Communications</em> 9, 577 (2018). [cited: 56]</a></p>
<p><a class="reference external" href="https://www.nature.com/articles/s41467-017-02177-w">Daniel B Lipka, Tania Witte, Reka Toth, Jing Yang, Manuel Wiesenfarth, Peter Nöllke, Alexandra Fischer, David Brocks, Zuguang Gu, <span class="underline">Jeongbin Park</span>, Brigitte Strahm, Marcin Wlodarski, Ayami Yoshimi, Rainer Claus, Michael Lübbert, Hauke Busch, Melanie Boerries, Mark Hartmann, Maximilian Schönung, Umut Kilik, Jens Langstein, Justyna A Wierzbinska, Caroline Pabst, Swati Garg, Albert Catalá, Barbara De Moerloose, Michael Dworzak, Henrik Hasle, Franco Locatelli, Riccardo Masetti, Markus Schmugge, Owen Smith, Jan Stary, Marek Ussowicz, Marry M Van Den Heuvel-Eibrink, Yassen Assenov, Matthias Schlesner, Charlotte Niemeyer, Christian Flotho, Christoph Plass, <strong>“RAS-pathway mutation patterns define epigenetic subclasses in juvenile myelomonocytic leukemia”</strong>, <em>Nature Communications</em> 8, 2126 (2017). [cited: 92]</a></p>
<p><a class="reference external" href="https://www.embopress.org/doi/abs/10.15252/msb.20177697">Julia Jabs, Franziska M Zickgraf, <span class="underline">Jeongbin Park</span>, Steve Wagner, Xiaoqi Jiang, Katharina Jechow, Kortine Kleinheinz, Umut H Toprak, Marc A Schneider, Michael Meister, Saskia Spaich, Marc Sütterlin, Matthias Schlesner, Andreas Trumpp, Martin Sprick, Roland Eils, Christian Conrad, <strong>“Screening drug effects in patient‐derived cancer cells links organoid responses to genome alterations”</strong>, <em>Molecular Systems Biology</em> 13, 955 (2017). [cited: 175]</a></p>
<p><a class="reference external" href="https://www.biorxiv.org/content/10.1101/199547v1.abstract">Daniel Hüebschmann, Nils Kurzawa, Sebastian Steinhauser, Philipp Rentzsch, Stephen Krämer, Carolin Andresen, <span class="underline">Jeongbin Park</span>, Roland Eils, Matthias Schlesner, Carl Herrmann, <strong>“Deciphering programs of transcriptional regulation by combined deconvolution of multiple omics layers”</strong>, <em>bioRxiv</em>, 199547 (2017). [cited: 8]</a></p>
<p><a class="reference external" href="https://www.sciencedirect.com/science/article/pii/S0021925820396034">Heon Seok Kim, Kyungjin Lee, Sangsu Bae, <span class="underline">Jeongbin Park</span>, Chong-Kyo Lee, Meehyein Kim, Eunji Kim, Minju Kim, Seokjoong Kim, Chonsaeng Kim, Jin-Soo Kim, <strong>“CRISPR/Cas9-mediated gene knockout screens and target identification via whole-genome sequencing uncover host genes required for picornavirus infection”</strong>, <em>Journal of Biological Chemistry</em> 292 (25), 10664-10671 (2017). [cited: 42]</a></p>
<p><a class="reference external" href="https://www.nature.com/articles/nmeth.4262"><span class="underline">Jeongbin Park</span>, Liam Childs, Daesik Kim, Gue-Ho Hwang, Sunghyun Kim, Sang-Tae Kim, Jin-Soo Kim, Sangsu Bae, <strong>“Digenome-seq web tool for profiling CRISPR specificity”</strong>, <em>Nature Methods</em> 14 (6), 548-549 (2017). [cited: 38]</a></p>
<p><a class="reference external" href="https://academic.oup.com/bioinformatics/article-abstract/33/2/286/2525673"><span class="underline">Jeongbin Park</span>, Kayeong Lim, Jin-Soo Kim, Sangsu Bae, <strong>“Cas-analyzer: an online tool for assessing genome editing results using NGS data”</strong>, <em>Bioinformatics</em> 33 (2), 286-288 (2017). [cited: 306]</a></p>
<p><a class="reference external" href="https://academic.oup.com/bioinformatics/article-abstract/32/13/2017/1743283"><span class="underline">Jeongbin Park</span>, Jin-Soo Kim, Sangsu Bae, <strong>“Cas-Database: web-based genome-wide guide RNA library design for gene knockout screens using CRISPR-Cas9”</strong>, <em>Bioinformatics</em> 32 (13), 2017-2023 (2016). [cited: 44]</a></p>
<p><a class="reference external" href="https://genome.cshlp.org/content/26/3/406.short">Daesik Kim, Sojung Kim, Sunghyun Kim, <span class="underline">Jeongbin Park</span>, Jin-Soo Kim, <strong>“Genome-wide target specificities of CRISPR-Cas9 nucleases revealed by multiplex Digenome-seq”</strong>, <em>Genome Research</em> 26 (3), 406-415 (2016). [cited: 207]</a></p>
<p><a class="reference external" href="https://academic.oup.com/bioinformatics/article-abstract/31/24/4014/198447"><span class="underline">Jeongbin Park</span>, Sangsu Bae, Jin-Soo Kim, <strong>“Cas-Designer: a web-based tool for choice of CRISPR-Cas9 target sites”</strong>, <em>Bioinformatics</em> 31 (24), 4014-4016 (2015). [cited: 324]</a></p>
<p><a class="reference external" href="https://www.nature.com/nmeth/journal/v12/n3/abs/nmeth.3284.html">Daesik Kim, Sangsu Bae, <span class="underline">Jeongbin Park</span>, Eunji Kim, Seokjoong Kim, Hye Ryeong Yu, Jinha Hwang, Jong-Il Kim, Jin-Soo Kim, <strong>“Digenome-seq: genome-wide profiling of CRISPR-Cas9 off-target effects in human cells”</strong>, <em>Nature Methods</em> 12 (3), 237-243 (2015). [cited: 981]</a></p>
<p><a class="reference external" href="https://academic.oup.com/bioinformatics/article-abstract/30/10/1473/267560">Sangsu Bae<sup>†</sup>, <span class="underline">Jeongbin Park</span><sup>†</sup>, Jin-Soo Kim, <strong>“Cas-OFFinder: a fast and versatile algorithm that searches for potential off-target sites of Cas9 RNA-guided endonucleases”</strong>, <em>Bioinformatics</em> 30 (10), 1473-1475 (2014). [cited: 1559]</a></p>
<p><a class="reference external" href="https://www.nature.com/articles/ncomms4724">Moon-Hyeong Seo, <span class="underline">Jeongbin Park</span>, Eunkyung Kim, Sungchul Hohng, Hak-Sung Kim, <strong>“Protein conformational dynamics dictate the binding affinity for a ligand”</strong>, <em>Nature Communications</em> 5, 3724 (2014). [cited: 142]</a></p>
</div>
<div class="text-right docutils container">
<div class="line-block">
<div class="line">Total Citations: 4383, H-index: 18</div>
<div class="line"><br /></div>
</div>
</div>
<div class="entries text-left docutils container">
<ul class="simple">
<li><p>†: Joint first authors.</p></li>
<li><p>The citation numbers were retrieved from Google Scholar.</p></li>
</ul>
</div>
<script>$("body").css("background", "#ddd");</script></section>
</section>


            <div class="clearer"></div>
          </div>
        </div>
      </div>


<script>$('.external').attr('target', '_blank');</script>

  </body>
</html>